.travis.yml

```diff
@@ -99,6 +99,12 @@ jobs:
       go: $GO_VERSION
       stage: test
 
+    # Journalbeat
```

Comment:
Can we have a follow up issue or meta issue to make sure we add a Job in Jenkins for it in the near future?

Reply:
I added one more element to the TODO list in the meta issue: https://github.com/elastic/beats/issues/8323